### PR TITLE
Implement listener for configure event

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
[Twitter Component listens [2]](https://trello.com/c/4Fx561gV/4642-twitter-component-listens-2)

File: test/unit/rise-twitter.spec.js [hunk 1]
```diff
@@ -1,10 +1,84 @@
...
+    risePlaylistItem.dispatchEvent(event);
+
+    setTimeout(()=>{
+      expect(component.screenName).toEqual("test");
+      done();
+    }, 1000);
+
+    jest.runAllTimers();
+  });
+
```
Comment (jungeunyoon) [inline test/unit/rise-twitter.spec.js]: @rodrigopavezi Small catch - all three of the following tests only test for `play` event.

Comment (rodrigopavezi) [reply test/unit/rise-twitter.spec.js]: @jungeunyoon ouch :sob: I will change. Thanks for the catch.

Comment (jungeunyoon) [reply test/unit/rise-twitter.spec.js]: @rodrigopavezi haha teamwork. looks good now 👍

**Pull request merged**